jupyterhub/orm.py

```diff
@@ -54,6 +56,38 @@ def process_result_value(self, value, dialect):
         return value
 
 
+class OptionalEncrypted(EncryptedType):
+    def __init__(self, type_in=None, key=None, engine=None, **kwargs):
+        try:
+            import cryptography
```

Comment:
Gotcha. Then maybe change the message to something like "auth_state will be unavailable". We'll probably also want to make sure there's a sensible error when auth_state is attempted to be set when it is not available.

One option is to do this at a slightly higher level, in a Python property on User and leave the column as raw bytes as far as sqlalchemy is concerned.